enterprise/frontend/src/embedding-sdk/config.ts

```diff
@@ -4,5 +4,5 @@ export const EMBEDDING_SDK_PORTAL_ROOT_ELEMENT_ID = "metabase-sdk-portal-root";
 export const EMBEDDING_SDK_FULL_PAGE_PORTAL_ROOT_ELEMENT_ID =
   "metabase-sdk-full-page-portal-root";
 
-export const getEmbeddingSdkVersion = () =>
-  process.env.EMBEDDING_SDK_VERSION ?? "unknown";
+export const getEmbeddingSdkVersion = (): string | "unknown" =>
```

Comment:
It was a bit unexpected that this function returned "unknown" instead of a falsy value, I wanted to make it clearer in the type